sources/components/custom-styles/template-cs-content.scss

```
@@ -185,5 +185,217 @@
+  */
+  #{--content-area-product-title__font-style}: normal;
+
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "text_decoration",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "underline",
+      "off": "none"
+    },
+    "icon": "underline",
+    "scope": "global"
+  */
+  #{--content-area-product-title__text-decoration}: none;
+
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "text_transform",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "uppercase",
+      "off": "none"
+    },
+    "icon": "uppercase",
+    "scope": "global"
+  */
+  #{--content-area-product-title__text-transform}: none;
+
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "color",
+    "type": "button",
+    "editor": "colorPicker",
+    "featured": true,
+    "scope": "global"
+  */
+  #{--content-area-product-title__color}: $black;
+
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+
+  #{--content-area-product-price__font-size}: 24px;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "font_size",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "700",
+      "off": "400"
+    },
+    "icon": "bold",
+    "scope": "global"
+  */
+
+  #{--content-area-product-price__font-weight}: 700;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "font_style",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "italic",
+      "off": "normal"
+    },
+    "icon": "italic",
+    "scope": "global"
+  */
+
+  #{--content-area-product-price__font-style}: normal;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "text_decoration",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "underline",
+      "off": "none"
+    },
+    "icon": "underline",
+    "scope": "global"
+  */
+
+  #{--content-area-product-price__text-decoration}: none;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "color",
+    "type": "button",
+    "editor": "colorPicker",
+    "featured": true,
+    "scope": "global"
+  */
+
+  #{--content-area-product-price__color}: #000;
+  /* VoogStyle
+    "pathI18n": ["product", "description"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+
+  #{--content-area-product-description__font-size}: 18px;
+  /* VoogStyle
+    "pathI18n": ["product", "description"],
+    "titleI18n": "font_size",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "700",
+      "off": "400"
+    },
+    "icon": "bold",
+    "scope": "global"
+  */
+
+  #{--content-area-product-description__font-weight}: 400;
```

It is too strong. Let's use 300 like in normal body.

```suggestion
  #{--content-area-product-description__font-weight}: 300;
```